### DISCUSSION TITLE
Custom database

### DISCUSSION THREAD

**Discussion by kisinga** [pocketbase/pocketbase#420] [Q&A]
Is it possible to completely strip out the database and use something else if I wanted, say mongo

**Comment by ganigeorgiev** [comment]
Mongo support is not planned. This was asked previously in https://github.com/pocketbase/pocketbase/discussions/58.

Support for other SQL databases may be added, but it is a low priority for now (see https://github.com/pocketbase/pocketbase/discussions/14#discussioncomment-3109447).

You could check our v1.0.0 roadmap here - https://github.com/orgs/pocketbase/projects/2

**Comment by kisinga** [comment]
@ganigeorgiev
What I mean is, is the code extensible enough that If I wanted to implement my own database implementation, in this Mongo, I could do it without having to break the project apart?

**Reply by ganigeorgiev** [reply]
>  is the code extensible enough that If I wanted to implement my own database implementation

No. The query library has abstraction for most sql operations but still there are some specifics that need to be tuned based on the used databases.
